.github/workflows/IOnIntegration.yml

```diff
@@ -0,0 +1,94 @@
+name: I-On Integration
+
+on:
```

Comment:
This workflow is triggered by more than one event. As a result, all the jobs in it are executed. The jobs push_integration and push_production are skipped under certain conditions, but all the checks are shown on pr page and commit summary. Do you see any inconvenience in having 3 workflows instead of one?

Reply:
It's a good point! I'll separate into 3 workflows

Reply:
If we split in 3 workflows and put each workflow in a separate file, we'll have to change more than one file if some aspect of how we build or deploy is altered. So having just one workflow could be simpler to maintain.